docs/docs/docs/01-core/content-releases/01-backend.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Backend Design
+description: Content Releases backend
+tags:
+  - content-releases
+  - tech design
+---
+
+All backend code can be found in:
+
+```
+ packages/core/content-releases/server
+```
+
+## Content-types
+
+The content-releases plugin creates two hidden content-types.
+
+### Release
+
+This content type stores all the information about a release and its associated Release Actions. It is saved in the database as `strapi_release`. The schema can be found in:
+
+```
+packages/core/content-releases/server/src/content-types/release/schema.ts
+```
+
+### Release Action
+
+This content type is associated with any entry from any content-type with draft and publish enable. It is also responsible for storing the action to perform for the associated entry. It is saved in the database as `strapi_release_actions`. The schema can be found in:
```

Comment:
this one has a plural db name, the other a singular one, I assume 1 of the 2 is wrong

Reply:
yeah they should both be plural 👍